lib/terminology.rb

```diff
@@ -25,6 +25,16 @@ class Terminology
     @@core_snomed = {}
     @@common_ucum = []
 
+    def self.reset
```

Comment:
Just a note that I added this because the new completeness_test was causing the terminology_test to fail, because it was loading the files before the terminology_test and then not loading the new dummy test files